app/controllers/chat_controller.rb

```diff
@@ -242,31 +242,32 @@ def flag
 
   def lookup_message
     chat_channel = @message.chat_channel
-    chatable = nil
     chatable = chat_channel.chatable
     guardian.ensure_can_see!(chatable)
     include_deleted = guardian.can_moderate_chat?(chatable)
-    base_query = ChatMessage
-      .includes(:in_reply_to)
-      .includes(:revisions)
-      .includes(:user)
-      .includes(chat_webhook_event: :incoming_chat_webhook)
-      .includes(reactions: :user)
-      .includes(:uploads)
-      .includes(chat_channel: :chatable)
-      .where(chat_channel: chat_channel)
-
+    base_query = ChatMessage.where(chat_channel: chat_channel)
     base_query = base_query.with_deleted if include_deleted
+
     past_messages = base_query
       .where("created_at < ?", @message.created_at)
-      .order(created_at: :desc).limit(40)
+      .order(created_at: :desc)
+      .limit(40)
 
     # .with_deleted if include_deleted
     future_messages = base_query
       .where("created_at > ?", @message.created_at)
```

Comment:
I can't remember the reasoning for using `created_at` instead of `id` for this - couldn't we grab any messages after using an ID greater than the `@message`? I assume that would be a PERF improvement

Reply:
I would prefer to investigate this in a different PR